util/rpcclient/rpcclient.go

```diff
@@ -56,6 +58,8 @@ var DefaultClientConfig = ClientConfig{
 	Retries:     3,
 	RetryErrors: "websocket: close.*|dial tcp .*|.*i/o timeout|.*connection reset by peer|.*connection refused",
 	ArgLogLimit: 2048,
+	// Use geth's unexported wsDefaultReadLimit from rpc/websocket.go
+	WebsocketMessageSizeLimit: 32 * 1024 * 1024,
```

Comment:
Let's increase this to 256 MiB (for the other websocket clients as well). All of these websocket clients are expecting the server to be honest.